utils/phase_utils.py

```diff
@@ -308,3 +310,29 @@ def add_usage_request(
         params=GetUsageParams(),
     )
     return [*operations, usage_request]
+
+
+def get_thinking_block(output: MiddlemanModelOutput) -> Optional[Dict[str, Any]]:
+    if not output.extra_outputs:
+        return None
+    return next(
+        (
+            block
+            for block in output.extra_outputs["content_blocks"]
+            if block["type"] == "thinking"
+        ),
+        None,
+    )
+
+
+def add_empty_user_turn(messages: List[Message]) -> List[Message]:
```

Comment:
```suggestion
def add_dummy_user_message(messages: List[Message]) -> List[Message]:
```

1. This is the only place in the codebase where the word "turn" is used
2. The `content` is not empty

Reply:
ah so the content can't be empty because Anthropic also complains about that